addons/l10n_jp/data/template/account.tax-jp.csv

```diff
@@ -1,49 +1,49 @@
-"l10n_jp_tax_sale_exc_8","1","8%","Sale (8%)","GST Sale 8%","percent","8.0","sale","False","True","l10n_jp_tax_group_8","base","invoice","+Taxable Sale Amount (8%)","","仮受消費税(外) 8%","仮受消費税(外) 8%"
+"l10n_jp_tax_sale_exc_8","1","8%","Sale (8%)","GST Sale 8%","percent","8.0","sale","False","True","l10n_jp_tax_group_8","base","invoice","+Taxable Sale Amount (8%)","","課税売上8%","課税売上8%"
 "","","","","","","","","","","","tax","invoice","+GST Sale 8%","l10n_jp_330800","",""
 "","","","","","","","","","","","base","refund","-Taxable Sale Amount (8%)","","",""
 "","","","","","","","","","","","tax","refund","-GST Sale 8%","l10n_jp_330800","",""
-"l10n_jp_tax_sale_exc_10","1","10%","Sale (10%)","GST Excluded Sale 10%","percent","10.0","sale","False","True","l10n_jp_tax_group_10","base","invoice","+Taxable Sale Amount (10%)","","仮受消費税(外) 10%","仮受消費税(外) 10%"
+"l10n_jp_tax_sale_exc_10","1","10%","Sale (10%)","GST Sale 10%","percent","10.0","sale","False","True","l10n_jp_tax_group_10","base","invoice","+Taxable Sale Amount (10%)","","課税売上10%","課税売上10%"
 "","","","","","","","","","","","tax","invoice","+GST Sale 10%","l10n_jp_330800","",""
 "","","","","","","","","","","","base","refund","-Taxable Sale Amount (10%)","","",""
 "","","","","","","","","","","","tax","refund","-GST Sale 10%","l10n_jp_330800","",""
-"l10n_jp_tax_sale_incl_8","1","8% Incl","GST Inc Sale (8%)","GST Inclusive Sale 8%","percent","8.0","sale","True","True","l10n_jp_tax_group_8","base","invoice","+Taxable Sale Amount (8%)","","仮受消費税(内) 8%","仮受消費税(内) 8%"
+"l10n_jp_tax_sale_incl_8","1","8% Incl","GST Inc Sale (8%)","GST Inclusive Sale 8%","percent","8.0","sale","True","True","l10n_jp_tax_group_8","base","invoice","+Taxable Sale Amount (8%)","","課税売上8% （税込）","課税売上8%（税込）"
 "","","","","","","","","","","","tax","invoice","+GST Sale 8%","l10n_jp_330800","",""
 "","","","","","","","","","","","base","refund","-Taxable Sale Amount (8%)","","",""
 "","","","","","","","","","","","tax","refund","-GST Sale 8%","l10n_jp_330800","",""
-"l10n_jp_tax_sale_incl_10","1","10% Incl","GST Inc Sale (10%)","GST Inclusive Sale 10%","percent","10.0","sale","True","True","l10n_jp_tax_group_10","base","invoice","+Taxable Sale Amount (10%)","","仮受消費税(内) 10%","仮受消費税(内) 10%"
+"l10n_jp_tax_sale_incl_10","1","10% Incl","GST Inc Sale (10%)","GST Inclusive Sale 10%","percent","10.0","sale","True","True","l10n_jp_tax_group_10","base","invoice","+Taxable Sale Amount (10%)","","課税売上10% （税込）","課税売上10%（税込）"
 "","","","","","","","","","","","tax","invoice","+GST Sale 10%","l10n_jp_330800","",""
 "","","","","","","","","","","","base","refund","-Taxable Sale Amount (10%)","","",""
 "","","","","","","","","","","","tax","refund","-GST Sale 10%","l10n_jp_330800","",""
 "l10n_jp_tax_sale_exempt","1","0% Exempt","GST Tax Exempt Sale","GST Tax Exempt Sale","percent","0.0","sale","False","True","l10n_jp_tax_group_exempt","base","invoice","-Tax Exempted Sale Amount","","輸出免税","輸出免税"

```

Comment:
```suggestion
"l10n_jp_tax_sale_exempt","1","0% Exempt","Export Tax Exempt Sale","Export Tax Exempt Sale","percent","0.0","sale","False","True","l10n_jp_tax_group_exempt","base","invoice","-Tax Exempted Sale Amount","","輸出免税","輸出免税"

```
Not sure it's relevant, but it's the only one that does not match the table in the spec :nerd_face:

Reply:
I didn't change it because:

We didn't plan to change the English version (although I changed some terms for clarity)
We also have a GST Tax Exempt Purchase so I think it's best to keep both aligned/